src/dashboard/src/media/js/ingest.js

```diff
@@ -295,25 +295,36 @@ $(function()
               });
             };
 
-          // redict to object/resource mapping pages
-          if ('- Upload DIP to Archivists Toolkit' == $select.find('option:selected').text())
-          {
-            $('body').html('<h1>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;' + gettext('Loading...') + '</h1>');
-            window.location.href = '/ingest/' + this.model.sip.get('uuid') + '/upload/atk/';
+          var selectedValue = $select.find('option:selected').text();
+
+          // If the Upload DIP targets a system where manual mapping is
+          // required, we forward the user to the corresponding page where
+          // we're going to collect the data required and continue the work.
+          //
+          // In other words, we're not going to execute the next job from
+          // JavaScript as we expect the pairing page to do it once the user
+          // has paired the items.
+          var dipUploadWithMappingPage = {
+            '- Upload DIP to ArchivesSpace':
+              '/ingest/' + this.model.sip.get('uuid') + '/upload/as/',
+            '- Upload DIP to Archivists Toolkit':
+              '/ingest/' + this.model.sip.get('uuid') + '/upload/atk/',
```

Comment:
I don't know how likely it is we need to handle a situation where the sip uuid isn't found? i.e. `this.model.sip.get('uuid')` but it seems we can avoid this repetition in calling `get(...)` twice, and increase the potential for handling that by assigning it to a variable first before assigning it in this object.

Reply:
I've declared the variable (`unitId`) as suggested but I don't know how I could handle and report the error. On one hand I don't think a backbone model would make it this far without an ID. But also I don't think there's logic in place at the moment to report the error to the user. What would you do?

```javascript
if (!this.model.sip.get('uuid')) {
    // Would you return?
    // Log the console?
    // ...?
}
```